deepeval/generator/question_generator_llama_index.py

```diff
@@ -0,0 +1,68 @@
+import os
+from llama_index import SimpleDirectoryReader, ServiceContext
```

Comment:
To manage complex external dependencies, can you please move these inside the `QuestionGenerator` abstraction?